src/platforms/eglstream-kms/server/kms_display_configuration.h

```diff
@@ -45,7 +45,7 @@ class KMSDisplayConfiguration : public DisplayConfiguration
     size_t get_kms_mode_index(DisplayConfigurationOutputId id, size_t conf_mode_index) const;
     void update();
 
-    void for_each_output(std::function<void(kms::EGLOutput const&)> const& f) const;
+    void for_each_output(std::function<void(std::shared_ptr<kms::EGLOutput> const&)> const& f) const;
```

Comment:
I think we should change the implementation, not the signature and all calling code

Reply:
On closer examination, ownership **is** being passed this way